src/DucksNet.Domain/Model/Employee.cs

```diff
@@ -67,27 +67,34 @@ public void AssignToOffice(Guid idOffice)
     {
         IdOffice = idOffice;
     }
-    public void UpdateFields(string surname, string firstName, string address, string ownerPhone, string ownerEmail)
+    public List<string> UpdateFields(string? surname, string? firstName, string? address, string? ownerPhone, string? ownerEmail)
```

Comment:
Why return a list of strings with errors here instead of a Result?

Reply:
because I forgot about this and I can not change it, there are too many modifications to be done. Next sprint.